src/cmd/flux-mini.py

```diff
@@ -34,10 +34,10 @@ def main():
     if sys.stderr.isatty():
         cmd = sys.argv[1] if len(sys.argv) > 1 else None
         if cmd in ["submit", "run", "batch", "alloc", "bulksubmit"]:
-            suggestion = f"flux {sys.argv[1]} "
```

Comment:
Also I'm wondering why the warning suggestion still showed `flux-batch` (with the hyphen) when "cmd" checks for the term without it.